### DISCUSSION TITLE
OPENJDK 11.0.2  that worked on my machine ( M1 Mac )

### DISCUSSION THREAD

**Discussion by venkatesan-framewirk** [Sunbird-Ed/Community#427] [Documentation]
The OpenJdk 11.0.2 that's available in this page, fixed my java problem.

https://jdk.java.net/archive/

**Comment by Krishnaj20** [comment]
thanks @venkatesan-framewirk 

@vinukumar-vs - Pls get this added to the microsite documentation.
